dotCMS/src/main/java/org/apache/velocity/runtime/VelocimacroManager.java

```diff
@@ -249,6 +249,13 @@ public VelocimacroProxy get(final String vmName, final String namespace, final S
         return null;
     }
 
+     public void dumpGlobalNamespace()
+     {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the end of previous line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ALeftCurlyBraceEndLineCheck)